### ISSUE TITLE
Firefox: NVDA does not properly read the search edit field on GitHub

### ISSUE THREAD

**Issue by bhavyashah** [nvaccess/nvda#13023]
Preface: This issue is probably broader in scope than the isolated instance I identify in this ticket.

### Steps to reproduce:
1 Open https://github.com/nvaccess/nvda/issues in Firefox.
2 Press "e" or otherwise navigate to the banner landmark    Site  search landmark    clickable  Search or jump to…  editable combo box.
3 Press Enter on it and type something.
4 Press NVDA+Tab and issue the read current line command (NVDA+L on laptop).

### Actual behavior:
For the former, NVDA says "unknown  defunct" and for the latter, NVDA says "blank."

### Expected behavior:
For the former, NVDA should announce the field label and its contents, and for the latter, NVDA should announce the field contents.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
NVDA 2021.3 Beta 1

#### Windows version:
Windows 10

#### Name and version of other software in use when reproducing the issue:
Firefox 94.0 

#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by lukaszgo1** [comment]
cc @jcsteh

**Comment by bhavyashah** [comment]
I no longer actively use Firefox. That said, I tried visiting the test page I provided in the issue description in Edge. If I remember correctly, there used to be two edit boxes on that page, of which the first one was the one I was referring to in the issue description. That edit field is no longer there and I don't have additional test cases. I'm happy for this ticket to be closed.

**Comment by Adriani90** [comment]
Yes for the current search field this issue is not reproducible. Closing as invalid. Thanks for the update.
